src/site/content/en/blog/web-otp/index.md

```diff
@@ -131,12 +132,11 @@ Let's say a user wants to verify their phone number with a website. The website
 sends a text message to the user over SMS and the user enters the OTP from the
 message to verify the ownership of the phone number.
 
-With the Web OTP API, these steps are as easy as one tap for the
-user, as demonstrated in the video. As the text message arrives, a bottom
-sheet pops up and prompts the user to verify their
-phone number. After clicking the **Verify** button within the bottom sheet,
-the browser pastes the OTP into the form and the form is submitted without
-the user needing to press **Continue**.
+With the Web OTP API, these steps are as easy as one tap for the user, as
+demonstrated in the video. When the text message arrives, a bottom sheet pops up
+and prompts the user to verify their phone number. After clicking the **Verify**
+button on the bottom sheet, the browser pastes the OTP into the form and the
+form is submitted without the user needing to press **Continue**.
 
 <video autoplay loop muted playsinline>
   <source src="https://storage.googleapis.com/web-dev-assets/sms-receiver-announce/demo.mp4" type="video/mp4">
```

Comment:
Let's replace the video. I can work on this.

Reply:
Thanks you.

Reply:
The video is replaced.